editor/src/components/canvas/dom-walker.ts

```diff
@@ -373,7 +390,7 @@ export function useDomWalker(props: CanvasContainerProps): React.Ref<HTMLDivElem
               }
             }
 
-            if (cachedMetadata == null) {
+            if (cachedMetadata == null || initComplete === false) {
```

Comment:
```suggestion
            if (cachedMetadata == null || !initComplete) {
```
*6 million dollar man eye zoom*

Reply:
Fixed.

But I think it's worth a quick vote. 
The dom walker function became rather complicated and harder to maintain in the last few months, and while changing it I felt (type) safer with the `=== false` check because if you change the type from boolean to something else (for example you wrap it in an array and forget to destructure) typescript will break on `=== true` but it will let you test the truthyness if you omit the `===`. 

OTOH, our (unwritten) style guide says that boolean checks should not compare to true or false.

What do you think?

Reply:
Do we still have no way to prevent truthiness? As that is the real underlying issue here.

Reply:
AFAIK there's no way to disallow truthiness in typescript

Reply:
Looks like we could do with using https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/strict-boolean-expressions.md